.github/workflows/go-security-checks.yml

```diff
@@ -35,9 +35,6 @@ jobs:
       - name: Install golangci-lint
         uses: golangci/golangci-lint-action@v6
         with:
-          # Needed to address this issue https://github.com/golangci/golangci-lint-action/issues/244
```

Comment:
Linter gave an error so I think this problem has been resolved. Also checked issue and it seems like its resolved.